webhook.sh

```diff
@@ -0,0 +1 @@
+curl -dH -X POST $WEBHOOK_URL 
```

Comment:
From memory I believe you can drop the `chmod` part of your command if you remember to include a shebang at the start of this script (e.g. `#!/bin/bash`) and worth noting that as long as curl gets a response it reports back a success; you may want to adjust this to add a `--fail` command so that it throws an error if a 4xx or 5xx status code is returned

Reply:
Note that this has hidden slightly the fact that Travis isn't triggering deployments successfully - e.g. your latest Travis run skipped deployment but the previous one had this output:
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"/>
<title>401 - Unauthorized: Access is denied due to invalid credentials.</title>
<style type="text/css">
<!--
body{margin:0;font-size:.7em;font-family:Verdana, Arial, Helvetica, sans-serif;background:#EEEEEE;}
fieldset{padding:0 15px 10px 15px;} 
h1{font-size:2.4em;margin:0;color:#FFF;}
h2{font-size:1.7em;margin:0;color:#CC0000;} 
h3{font-size:1.2em;margin:10px 0 0 0;color:#000000;} 
#header{width:96%;margin:0 0 0 0;padding:6px 2% 6px 2%;font-family:"trebuchet MS", Verdana, sans-serif;color:#FFF;
background-color:#555555;}
#content{margin:0 0 0 2%;position:relative;}
.content-container{background:#FFF;width:96%;margin-top:8px;padding:10px;position:relative;}
-->
</style>
</head>
<body>
<div id="header"><h1>Server Error</h1></div>
<div id="content">
 <div class="content-container"><fieldset>
  <h2>401 - Unauthorized: Access is denied due to invalid credentials.</h2>
  <h3>You do not have permission to view this directory or page using the credentials that you supplied.</h3>
 </fieldset></div>
</div>
</body>
</html>
```

The structure here looks reasonable, so the most likely cause is if we haven't escaped the environment variable properly -  I think you might have seen this issue before but if not then worth checking [the Travis docs on escaping](https://docs.travis-ci.com/user/encryption-keys/#note-on-escaping-certain-symbols) and do shout if nothing seems to be playing ball!

Reply:
resolved issue with env